data/dock.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Pantheon Dock
+
+After=io.elementary.gala.target
+
+Requisite=gnome-session-initialized.target
+PartOf=gnome-session-initialized.target
+Before=gnome-session-initialized.target
```

Comment:
```suggestion
PartOf=graphical-session.target
After=gnome-session-initialized.target
```

Otherwise, dock is initialized before a display is set.